WordPress/dashicons
# Build: refactoring grunt for clearer folder structure, tweaks

This PR refactor the repository for clarity:

1. All source files are now in `sources/`, svg sources in `sources/svg/`
2. ~Adds PHP library build~ (see below)
3. Clarified gruntfile comments
4. All the builds are based on `svg-min` now instead of `svg`
5. Removed titles for accessibility reasons.

This aligns the build script to: https://github.com/Automattic/gridicons/pull/217

## To test

1. Clear the repository (ideally clone the repository again)
2. Try to reinstall / rebuild everything `npm install` + `npm run build`
3. Check the files built with this PR match the files before this PR
